CHANGELOG.md

```diff
@@ -17,15 +17,17 @@
 
 ### Added
 
-- Added same site property to the clear cookies function ([#218](https://github.com/chingu-x/chingu-dashboard-be/pull/218))
-- Added routes for teams to create own tech stack categories([#208](https://github.com/chingu-x/chingu-dashboard-be/pull/208))
-- Added unit tests for Features controller and services ([#220](https://github.com/chingu-x/chingu-dashboard-be/pull/220))
-- Added GET endpoint for solo project ([#223](https://github.com/chingu-x/chingu-dashboard-be/pull/223))
+-   Added same site property to the clear cookies function ([#218](https://github.com/chingu-x/chingu-dashboard-be/pull/218))
+-   Added routes for teams to create own tech stack categories([#208](https://github.com/chingu-x/chingu-dashboard-be/pull/208))
+-   Added unit tests for Features controller and services ([#220](https://github.com/chingu-x/chingu-dashboard-be/pull/220))
+-   Added GET endpoint for solo project ([#223](https://github.com/chingu-x/chingu-dashboard-be/pull/223))
+-   Updated readme for installation part ([#225])(https://github.com/chingu-x/chingu-dashboard-be/pull/225)
```

Comment:
Should probably be in the `### Changed` section.

If you can please also try to revert all the formatting changes. Thanks!

Reply:
I have moved it but I can't see the formatting changes in my ide, for me it looks normal....:(

Reply:
https://github.com/chingu-x/chingu-dashboard-be/pull/225/files

You can see in this page, looks like a space or two is added in each line

Reply:
@andram11 I had issues with this too. I don't really know how to fix it, but the easiest solution for me was to manually update the changelog directly from the repo